validating_admission_policies/cli_test/kyverno-test.yaml

```diff
@@ -0,0 +1,16 @@
+name: check_deployment_replicas
+policies:
+  - check_deployment_replicas.yaml
+resources:
+  - deployment.yaml
+results:
+  - policy: check-deployment-replicas
+    rule: object.spec.replicas <= 5
+    resource: nginx-deployment-1
+    kind: Deployment
+    result: pass
+  - policy: check-deployment-replicas
+    rule: object.spec.replicas <= 5
+    resource: nginx-deployment-2
+    kind: Deployment
+    result: pass
```

Comment:
It looks like this result should be a `fail` because `nginx-deployment-2` has 7 replicas.

Reply:
yes, the output should be as follows:
```
Executing check_deployment_replicas...

│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ # │ POLICY                    │ RULE                      │ RESOURCE                       │ RESULT │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ 1 │ check-deployment-replicas │ object.spec.replicas <= 5 │ /Deployment/nginx-deployment-1 │ Pass   │
│ 2 │ check-deployment-replicas │ object.spec.replicas <= 5 │ /Deployment/nginx-deployment-2 │ Fail   │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│

Test Summary: 1 tests passed and 1 tests failed

Aggregated Failed Test Cases : 
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ # │ POLICY                    │ RULE                      │ RESOURCE                       │ RESULT │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ 2 │ check-deployment-replicas │ object.spec.replicas <= 5 │ /Deployment/nginx-deployment-2 │ Fail   │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
```

In case I changes `result` to `fail`, the output should be as follows:
```
Executing check_deployment_replicas...

│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ # │ POLICY                    │ RULE                      │ RESOURCE                       │ RESULT │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│
│ 1 │ check-deployment-replicas │ object.spec.replicas <= 5 │ /Deployment/nginx-deployment-1 │ Pass   │
│ 2 │ check-deployment-replicas │ object.spec.replicas <= 5 │ /Deployment/nginx-deployment-2 │ Pass   │
│───│───────────────────────────│───────────────────────────│────────────────────────────────│────────│

Test Summary: 2 tests passed and 0 tests failed
```

Reply:
Yes, that is the expected results. Tests should always pass. When the expected results matches the observed result it is scored as a pass. When there's a mismatch it's a fail.

Reply:
I got your point. I updated it.